client/reader/get-helpers.js

```diff
@@ -160,3 +160,27 @@ export const canBeMarkedAsSeen = ( { post = null, posts = [], currentRoute = ''
 
 	return false;
 };
+
+/**
+ * Return Featured image alt text.
+ *
+ * @param {Object} post object containing post information
+ *
+ * @returns {string} Featured image alt text
+ */
+export const getFeaturedImageAlt = ( post ) => {
+	// Each post can have multiple images attached. To make sure we are selecting
+	// the alt text of the correct image attachment, we get the ID of the post thumbnail first
+	// and then use it to get the alt text of the Featured image.
+	const postThumbnailId = get( post, 'post_thumbnail.ID' );
+	const featuredImageAlt = get( post, 'attachments.' + postThumbnailId + '.alt' );
+	const postTitle = post.title;
```

Comment:
In an attempt to phase out lodash whenever we can, we can do

```suggestion
	// and then use it to get the alt text of the Featured image.
	const postTitle = post.title;
	const postThumbnailId = post?.post_thumbnail?.ID;
	const featuredImageAlt = post?.attachments[postThumbnailId]?.alt;	
```

Reply:
> In an attempt to phase out lodash whenever we can, we can do

Thanks Omar! That's a great TIL. I felt that using `get()` was a bit clumsy, but didn't know about `?.`.

I have fixed that now in the following commit: https://github.com/Automattic/wp-calypso/pull/55378/commits/fb317dced040699bb86ea2412e536dd001751adc